package.json

```diff
@@ -1,92 +1,101 @@
 {
   "name": "mongodb-vscode",
-  "displayName": "MongoDB",
-  "description": "MongoDB for Visual Studio Code",
+  "displayName": "MongoDB for VS Code",
+  "description": "Connect to MongoDB and Atlas and directly from your VS Code environment, navigate your databases and collections, inspect your schema and use playgrounds to prototype queries and aggregations.",
   "version": "0.0.1",
-  "engines": {
-    "vscode": "^1.40.0",
-    "node": "^12.4.0",
-    "npm": ">=6.13.0"
+  "homepage": "https://github.com/mongodb-js/vscode",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mongodb-js/vscode"
+  },
+  "bugs": {
+    "url": "https://github.com/mongodb-js/vscode/issues",
+    "email": "vscode@mongodb.com"
   },
+  "publisher": "mongodb",
   "categories": [
+    "Programming Languages",
+    "Snippets",
     "Other"
   ],
-  "activationEvents": [
-    "onCommand:mdb.connect",
-    "onCommand:mdb.connectWithURI",
-    "onCommand:mdb.disconnect",
-    "onCommand:mdb.removeConnection",
-    "onCommand:mdb.openMongoDBShell",
-    "onCommand:mdb.createPlayground",
-    "onView:mongoDB",
-    "onLanguage:json"
+  "keywords": [
+    "MongoDB",
+    "MongoDB Atlas",
+    "MQL",
+    "Database"
   ],
-  "main": "./out/extension.js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/mongodb-js/vscode"
+  "preview": true,
+  "icon": "resources/mongodb.png",
+  "galleryBanner": {
+    "color": "#21313D",
+    "theme": "dark"
   },
```

Comment:
@fredtruman can you take a look at this stuff? Documentation here: https://code.visualstudio.com/api/references/extension-manifest#marketplace-presentation-tips

In short:
- `icon`: the icon that is displayed in the marketplace (the word count icon in [this example](https://marketplace.visualstudio.com/items?itemName=ms-vscode.wordcount)
- `color`: the color of the header in the marketplace (the red in [this example](https://marketplace.visualstudio.com/items?itemName=ms-vscode.wordcount))

Reply:
Hey @mmarcon - recommending this:
<img width="1182" alt="Screenshot 2020-03-05 14 00 18" src="https://user-images.githubusercontent.com/489217/76015758-ab25e600-5ee9-11ea-8c69-3f4f9d5a7324.png">

**Assets and color specs:**
Banner background:
#3D4F58

Button:
Static: #13AA52
Hover: #0B3B35

Icon for header:
![mongodb-icon](https://user-images.githubusercontent.com/489217/76015868-dc9eb180-5ee9-11ea-8f4a-e88caa9c366e.png)

Reply:
@fredtruman I might be wrong but I don't think we can control the color of the button. Looks like it's the same for all the extensions.

Static: #107c10
Hover: #AAAAAA

Reply:
Ah gotcha. Ok I think this gray background still works with the existing/standard button colors